modules/Bio/EnsEMBL/Production/Pipeline/MirrorLoad/ListDatabase.pm

```diff
@@ -73,7 +81,7 @@ sub run {
 	}
 
 	#list all functional dbs
-	if( !( $self->param('process_grch37') || $self->param('process_mart') ) || $self->param('run_all') ){
+	if( !( $self->param('process_grch37') || $self->param('process_mart') || $self->param('process_grch37_mart') ) || $self->param('run_all') ){
```

Comment:
imho, Don't know the actual impact, but since this is same target for `all` grch37, I would just have remove the mart filter in case of grch37.

Reply:
added grch37 mart  flag to give more flexibility to copy only functional dbs or mart dbs for grch37

Reply:
added grch37 mart flag to give more flexibility to copy only functional or mart dbs for grch37

Reply:
Not sure this is a real use case, we'll see with usage :-)